### ISSUE TITLE
InfoNCE

### ISSUE THREAD

**Issue by TheTamT3** [perceptiveshawty/RankCSE#2]
In the InfoNCE loss,  the expression (in log function) is the softmax?

**Comment by perceptiveshawty** [comment]
I'm not sure I fully understand the question, if you're referring to nn.CrossEntropy(), the inputs are pre-softmax logits obtained by temperature-scaling the pairwise similarities

From PyTorch documentation on nn.CrossEntropy(): "The input is expected to contain the unnormalized logits for each class (which do not need to be positive or sum to 1, in general)."
